src/calculateRentalCost.js

```diff
@@ -21,6 +21,13 @@
  */
 function calculateRentalCost(days) {
   // write code here
+  if (days >= 7) {
+    return days * 40 - 50;
+  } else if (days >= 3) {
+    return days * 40 - 20;
+  } else {
```

Comment:
no need for this else
```suggestion
  }
```